distribution/src/main/assembly/assembly-openshift-drools-hacep.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+
+  <id>assembly-openshift-drools-hacep</id>
+  <formats>
+    <format>dir</format>
+    <format>zip</format>
+  </formats>
+
+  <includeBaseDirectory>true</includeBaseDirectory>
+
+  <fileSets>
+    <fileSet><!-- Note: going outside the module dir is bad, but it is not fetching generated files -->
+      <includes>
+        <include>../LICENSE-Apache-2.0.txt</include>
+      </includes>
+      <outputDirectory/>
+    </fileSet>
+    <!-- Examples -->
+    <fileSet><!-- Note: going outside the module dir is bad, but it is not fetching generated files -->
+      <directory>..</directory>
+      <outputDirectory>sources</outputDirectory>
+      <excludes>
+        <exclude>**/target/**</exclude>
+        <exclude>**/local/**</exclude>
+        <exclude>.*/**</exclude>
+        <exclude>nbproject/**</exclude>
+        <exclude>*.ipr</exclude>
+        <exclude>*.iws</exclude>
+        <exclude>**/*.iml</exclude>
+        <exclude>.git/**</exclude>
+        <exclude>/distribution/**</exclude>
+        <exclude>/doc/**</exclude>
+        <exclude>/integration/**</exclude>
+        <exclude>/jdkhttp/**</exclude>
+        <exclude>/sample-hacep-project/**</exclude>
+        <exclude>/thorntail/**</exclude>
+      </excludes>
+    </fileSet>
+  </fileSets>
+
+  <dependencySets>
+    <dependencySet>
+      <includes>
+        <include>org.kie:openshift-kie-hacep:jar</include>
+        <include>org.kie:kie-remote:jar</include>
+        <include>org.kie:openshift-kie-springboot:jar</include>
```

Comment:
SpringBoot jar is the only self contained executable that has sense to be included here.
Kie-remote and hacep jars require additional dependencies that should be included too. This will probably increase a lot dist zip file so I suggest for now to remove them and then in the future publish them via maven repo